test/e2e/specs/site-editor/page-list.spec.js

```diff
@@ -53,4 +61,356 @@ test.describe( 'Page List', () => {
+					await mediaLibrary
+						.locator( '.media-frame-toolbar' )
+						.waitFor( {
+							state: 'hidden',
+						} );
+
+					await mediaLibrary
+						.getByRole( 'button', { name: 'Select', exact: true } )
+						.click();
+				},
+				assertInitialState: async ( page ) => {
+					const el = page.getByText( 'Choose an image…' );
+					const placeholder = page.getByRole( 'button', {
+						name: 'Choose an image…',
+					} );
+					await expect( el ).toBeVisible();
+					await expect( placeholder ).toBeVisible();
+				},
+				assertEditedState: async ( page ) => {
+					const placeholder = page.getByRole( 'button', {
+						name: 'Choose an image…',
+					} );
+					await expect( placeholder ).toBeHidden();
+					const img = page.locator(
+						'.fields-controls__featured-image-image'
+					);
+					await expect( img ).toBeVisible();
+				},
+			},
+			statusVisibility: {
+				performEdit: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await statusAndVisibility.click();
+					const options = [
+						'Published',
+						'Draft',
+						'Pending Review',
+						'Private',
+					];
+
+					for ( const option of options ) {
+						await page
+							.getByRole( 'radio', { name: option } )
+							.click();
+						await expect( statusAndVisibility ).toContainText(
+							option
+						);
+
+						if ( option !== 'Private' ) {
+							await page
+								.getByRole( 'checkbox', {
+									name: 'Password protected',
+								} )
+								.check();
+						}
+					}
+				},
+				assertInitialState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Published'
+					);
+				},
+				assertEditedState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Private'
+					);
+				},
+			},
+			author: {
+				assertInitialState: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await expect( author ).toContainText( 'admin' );
+				},
+				performEdit: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await author.click();
+					const selectElement = page.locator(
+						'select:has(option[value="1"])'
+					);
+					await selectElement.selectOption( { value: '1' } );
+				},
+				assertEditedState: async () => {},
+			},
+			date: {
+				assertInitialState: async ( page ) => {
+					const dateEl = page.getByLabel( 'Edit Date' );
+					const date = new Date();
+					const yy = String( date.getFullYear() );
+
+					await expect( dateEl ).toContainText( yy );
+				},
+				performEdit: async ( page ) => {
+					const dateEl = page.getByLabel( 'Edit Date' );
+					await dateEl.click();
+					const date = new Date();
+					const yy = Number( date.getFullYear() );
```

Comment:
Nit: for a moment, my mind read this as if the year was two-digits — which is not what we render on screen. Naming this year or yyyy would be clearer.